src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -203,11 +203,17 @@
     private String configHtmlFile;
 
     /**
-     * File or classpath location of a properties file to use in xml formatting.
+     * File or classpath location of a properties file to use in jsoup xml formatting.
      */
     @Parameter(defaultValue = "formatter-maven-plugin/jsoup/xml.properties", property = "configxmlfile", required = true)
     private String configXmlFile;
 
+    /**
+     * File or classpath location of a properties file to use in eclipse xml formatting.
+     */
+    @Parameter(defaultValue = "formatter-maven-plugin/eclipse/xml.properties", property = "configexmlfile", required = true)
+    private String configEXmlFile;
```

Comment:
Using the Eclipse XML formatter makes a lot more sense to me, since this project's main purpose is to automate the application of Eclipse formatting rules. I don't know much about jsoup, but I've yet to find its formatting useful.

Rather than create a new formatter option like this, I can think of two possible alternatives:

1. This replaces the jsoup implementation
2. We do something like what I proposed in #254 to create separate mojos for each formatter, rather than overload the one

Reply:
Yes, I think Eclipse's formatting is much more useful thatn jsoup's, I only left it as default in order to prvent breaking existing users. So if you're ok with a replacement I'm 100% good with that.

Reply:
Totally agree Jsoup solution was never actually useful.  What was really wanted was what we could do in Eclipse directly and that was a 'beta' type attempt to get something that eventually might evolve so if we have a better solution, jsoup certain can go.  I would highly doubt anyone is actually using the jsoup soultion provided here as it is almost entirely not readable at a human level how we would expect things.  So I don't think it even warrents deprecation but straight replacement.

Reply:
OK. If we resolve the other issues raised I'll change the PR to replace jsoup entirely.